packages/orchestration/src/exos/remote-chain-facade.js

```diff
@@ -83,30 +84,27 @@ const prepareRemoteChainFacadeKit = (
     },
     {
       public: {
+        /** @type {HostOf<Chain['getChainInfo']>} */
         getChainInfo() {
           return watch(this.state.remoteChainInfo);
         },
 
-        /** @returns {Vow<PromiseToVow<OrchestrationAccount<ChainInfo>>>} */
+        /** @type {HostOf<Chain['makeAccount']>} */
         makeAccount() {
-          // TODO #9449 fix types
-          // @ts-expect-error Type 'Vow<Voidless>' is not assignable to type 'Vow<OrchestrationAccountI>'
-          return asVow(() => {
-            const { remoteChainInfo, connectionInfo } = this.state;
-            const stakingDenom = remoteChainInfo.stakingTokens?.[0]?.denom;
-            if (!stakingDenom) {
-              throw Fail`chain info lacks staking denom`;
-            }
+          const { remoteChainInfo, connectionInfo } = this.state;
+          const stakingDenom = remoteChainInfo.stakingTokens?.[0]?.denom;
+          if (!stakingDenom) {
+            return asVow(Fail`chain info lacks staking denom`);
```

Comment:
This will not do what you expect, and throw an error instead of returning a rejected vow

Reply:
should have been this?
```suggestion
            return asVow(() => Fail`chain info lacks staking denom`);
```

Reply:
That would work but I'm not sure why this PR removed the surrounding `asVow` in the first place.

Reply:
I the assume bug is that  `Fail` throws immediately here instead of propoagating as a vow?

What's a good way to catch this in a test? When reviewing this, I made similar edits to [cosmos-orch-acct](https://github.com/Agoric/agoric-sdk/blob/24528f1e377f41e13e9fe47a6f04c0f4b0c33452/packages/orchestration/src/exos/cosmos-orchestration-account.js#L445-L465) and didn't see any of these tests break:
https://github.com/Agoric/agoric-sdk/blob/24528f1e377f41e13e9fe47a6f04c0f4b0c33452/packages/orchestration/test/examples/stake-ica.contract.test.ts#L246-L261
The above does not go through the membrane, though. Would the membrane catch this? _On my short list to test._

Reply:
> didn't see any of these tests break

yeah, we need breaking tests. Addressing in https://github.com/Agoric/agoric-sdk/pull/9696